crates/interledger-stream/src/packet.rs

```diff
@@ -868,23 +884,28 @@ fn saturating_read_var_uint<'a>(reader: &mut impl BufOerExt<'a>) -> Result<u64,
 #[cfg(test)]
 mod fuzzing {
     use super::{StreamPacket, StreamPacketBuilder};
+    use bytes::{Buf, BytesMut};
 
     #[test]
-    #[ignore]
-    fn fuzzed_0_extra_trailer_bytes() {
-        // junk trailer mentioned in RFC might be applicable here, see
-        // fuzzed_1_unknown_frames_dont_roundtrip
-        roundtrip(&[
-            1, 14, 3, 19, 5, 3, 1, 14, 3, 0, 0, 0, 0, 14, 3, 0, 14, 5, 17,
-        ]);
+    #[cfg(feature = "strict")]
+    fn fuzzed_0_varint_length_prefix_with_leading_zeros_should_error() {
+        #[rustfmt::skip]
+        let input: &[u8] = &[
+            // Version, packet type, sequence and prepare amount
+            1, 14, 3, 19, 5, 3, 1, 14,
+            // len + num frame (which is zero)
+            3, 0, 0, 0,
```

Comment:
```suggestion
            1, 0,
```

With this it'll `roundtrip(input);` as discussed and the `#[cfg(feature = "strict")]` can be removed and the name changed.

Reply:
Aah now I see you added the case 3 for this one, my bad. I'd be inclined to only save one of these, since this has been fixed in the `interledger-packet` even though it's likely this was discovered through `interledger-stream`?

Reply:
Yes, you are correct that this was fixed in `interledger-packet`.  I think this was kept just so that the actual error was clarified and not mixed in with trailer bytes. Will remove.